Finalizing (#130)
greenelab/iscb-diversity-manuscript: content/20.results.md

Using citation counts has key limitations: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
Furthermore, the act of being honored, particularly with a keynote at an international meeting, could lead to work being more recognized and cited, which would reverse the arrow of causality.
We expected this approach to be overly conservative, leading to an acceptance of the null hypothesis of no disparities even if disparities existed.

> We present this version of the results in this paper, referring to supplementary figure S1 for other versions of the analyses.

Reviewer: ```suggestion
We present this version of the results in this paper.
Parallel analyses for the other versions are available in supplementary figure S1.
```
